src/libraries/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -130,6 +130,19 @@ public async Task WriteFrameAsync(Frame frame, CancellationToken cancellationTok
             await _connectionStream.WriteAsync(writeBuffer, 0, writeBuffer.Length, cancellationToken).ConfigureAwait(false);
         }
 
+        public async Task WriteFramesAsync(Frame[] frames, CancellationToken cancellationToken = default)
```

Comment:
This is just to make the test more likely to fail without the fix, right?

Reply:
yes